code/modules/hydroponics/grown.dm

```diff
@@ -117,11 +119,18 @@
 	if(seed)
 		for(var/datum/plant_gene/trait/trait in seed.genes)
 			trait.on_squash(src, target)
-
-	reagents.reaction(T)
-	for(var/A in T)
-		reagents.reaction(A)
-
+	if(!seed.get_gene(/datum/plant_gene/trait/noreact))
+		reagents.reaction(T)
+		for(var/A in T)
+			reagents.reaction(A)
+		qdel(src)
+	if(seed.get_gene(/datum/plant_gene/trait/noreact))
+		playsound(loc, 'sound/effects/fuse.ogg', 75, 1, -3)
+		addtimer(CALLBACK(src, .proc/prime), rand(30, 60))

```

Comment:
Given how I've heard these plants aren't easy to pull off, I would suggest changing the range to 20, 50

Reply:
sepchems aint too hard to get, it's the high death mixes that are (and are still rushable in a half hour or so by a competent botanist)

a 2 second fusetime is a liability to the user, however, at least if you use it like a grenade